app/commands/track/unimplemented_practice_exercises.rb

```diff
@@ -0,0 +1,15 @@
+class Track::UnimplementedPracticeExercises
```

Comment:
At some point, it might be nice to show the unimplemented exercises.
Something for a follow-up PR I feel

Reply:
Commands should be verbs.

```suggestion
class Track::RetrieveUnimplementedPracticeExercises
```